crates/ironrdp-pdu/src/codecs/rfx/header_messages.rs

```diff
@@ -171,10 +171,7 @@ pub struct RfxChannelWidth(i16);
 
 impl RfxChannelWidth {
     pub fn new(value: i16) -> Result<Self, RfxError> {
-        (1..=4096)
```

Comment:
Are those were accidentally removed lines?

Reply:
These limits are "SHOULD"s in the [spec](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdprfx/4060f07e-9d73-454d-841e-131a93aca675#Appendix_A_Target_3)

<img width="880" alt="image" src="https://github.com/Devolutions/IronRDP/assets/13578537/92366d5b-5d84-42be-93d7-1635da4f758e">

and the footnote says this:

<img width="625" alt="image" src="https://github.com/Devolutions/IronRDP/assets/13578537/0e5cbefe-550c-4406-afd6-b712b84c4530">

In practice these limits were constraining the screen to a relatively small size. I haven't found a limit that the server won't serve us, e.g. I just tried `width: 5120, height: 2774` (fullscreen on my 32" monitor without any scale factor), the performance was terrible but the server will send it. (With a proper scale factor, fullscreen works fine).

Reply:
In that case, you can remove the `Result` from the signature